_translator-files/README.md

```diff
@@ -1,22 +1,23 @@
 # Website docs translation:
 
-The recommended way to translate the website documentation is by using [Weblate](https://hosted.weblate.org/projects/jamulus/). If you are comfortable using GitHub and .po file editors, and still want to contribute your translation via a Pull Request, the files for each language are contained in: `_translator-files/po/LANGUAGE/`. You can find a guide on .po file editors [here](https://jamulus.io/kb/2022/08/04/Translating-po-files.html). Please make your PR against the `next-release` branch.
+The recommended way to translate the website documentation is by using [Weblate](https://hosted.weblate.org/projects/jamulus/). If you don't already have one, we recommend that you create a GitHub account and log in to Weblate using that. If you are comfortable using GitHub and .po file editors, and still want to contribute your translation via a pull request, the files for each language are contained in: `_translator-files/po/LANGUAGE/`. You can find a guide on .po file editors [here](https://jamulus.io/kb/2022/08/04/Translating-po-files.html). Please make your PR against the `next-release` branch.
+
+All the Jamulus project files (application and website) are listed together on Weblate. The files ("components") corresponding to the application itself are "Jamulus app" and "Windows Installer". All the rest correspond to the website.
 
 **Some tips regarding translations:**
 
 - Change the `lang: en` parameter in the files containing it at the beginning of the file to your language code (it, fr, de, etc.). Remember that if you are translating a regional variation (e.g. `pt_BR`), that's what you must use. Also make sure the `permalink: [parameter]` (e.g. /wiki/Client-Troubleshooting) stays the same
-- At times there are links to other docs `[translate this](don't translate this)` or specific paragraphs in other parts of the website: `[translate this](don't translate this#translate-this-but-don't-use-capitals)`. Make sure the last part uses the exact same words as the relevant paragraph title you use in your translation.
+- At times there are links to other docs: `[translate this](don't translate this)`, or specific paragraphs in other parts of the website: `[translate this](don't translate this#translate-this-but-don't-use-capitals)`. Make sure the last part after the hashtag uses the exact same words as the relevant paragraph title you use in your translation.
```

Comment:
It's probably worth _real_ examples here.

Reply:
Hmm, I'm not particularly smart but I had to read your example several times before I could understand it. My reasoning is that the examples look exactly like they will when translators come across them. These links have caused no end of confusion in the past and I think that after doing a few you become familiar with how they're done - and what each part is.

BTW, I added the fact that apostrophes shouldn't be used either.

Reply:
Reading it a second time, it seems fine now.